project_devops/gitbasics.txt

```diff
@@ -0,0 +1,19 @@
+git config --global user.name "username"
+git config --global user.email "email"
+git init
```

Comment:
Why did you wite this command

Reply:
To revise daily to get perfection in commands

Reply:
Good.